ard_controller/ard_controller.ino

```diff
@@ -38,9 +47,24 @@ struct JointAngleStruct {
 
 JointAngleStruct temporary_packet_data;
 JointAngleStruct validated_packet_data;
+JointAngleStruct joint_angle_goal;
+
+HardwareSerial& odrv_leftleg_ser = Serial2;
+HardwareSerial& odrv_rightleg_ser = Serial3;
+HardwareSerial& odrv_leftarm_ser = Serial4;
+HardwareSerial& odrv_rightarm_ser = Serial5;
```

Comment:
Cool, I'm assuming this is how the robot is set up?

Reply:
@agupta231 These statements just rename the hardware serial ports to match what they're communicating with (different odrives in this case)

@mahajanrevant  They're defined in the Teensy's board definition package. It's the same thing as `Serial` just existing for pretty much all arduinos, but since the Teensy has multiple, they added Serial1, Serial2, Serial3, ..., Serial8. I can add a comment about it in the code.